backend/app/api/deps.py

```diff
@@ -53,11 +70,15 @@ async def gidgethub_instance(request: Request):
 
 async def github_api_instance(
     request: Request, gh: gidgethub.aiohttp.GitHubAPI = Depends(gidgethub_instance)
-):
+) -> GithubAPI:
     session_data = fastapi_sessions.get_session(request)
     github_api = GithubAPI(
         gh=gh,
         username=session_data.github_user_id,
         default_role=app_state.settings.github.organisation_membership,
     )
     return github_api
+
+
+async def external_url_api_instance() -> ExternalUrlAPI:
```

Comment:
I like this approach of passing parts of app_state in from here, would be good to use this pattern as much as is practical.